WORKSPACE

```diff
@@ -16,6 +16,18 @@ android_ndk_repository(
     api_level = 14,
 )
 
+git_repository(
```

Comment:
These dependencies are outdated, please update

Reply:
Resolved in https://github.com/iotaledger/entangled/pull/26/commits/84cebe5a093f98dfbf68040ed69e5ef06b2af9db